firefox-ios/Shared/Strings.swift

```diff
@@ -2197,6 +2197,12 @@ extension String {
             value: "Summarized by Apple Intelligence",
             comment: "The label displayed in the summary report when the summary was generated using Apple Intelligence."
         )
+        public static let FootnoteLabel = MZLocalizedString(
+            key: "Summarizer.Footnote.Label.v142",
+            tableName: "Summarizer",
+            value: "Note: Summarization can make errors.",
```

Comment:
not sure if we do anything special with `:` like we do with `...` but we can see during strings PR. I couldn't find anything in the docs. cc: @Delphine  :D

Reply:
I just checked in the `Strings.swift` and we don't do anything special we are using `:` in other places. @Delphine can confirm though 😄

Reply:
welp, we will find out lol 😄 will merge this so we can get changes in for the build and can do a follow up ticket.